superchain/configs/sepolia/arena-z-testnet.toml

```diff
@@ -18,6 +18,7 @@ max_sequencer_drift = 600
   fjord_time = 0 # Thu 1 Jan 1970 00:00:00 UTC
   granite_time = 1723478400 # Mon 12 Aug 2024 16:00:00 UTC
   holocene_time = 1732633200 # Tue 26 Nov 2024 15:00:00 UTC
+  pectra_blob_schedule_time = 1776441600 # Fri 17 Apr 2026 16:00:00 UTC
```

Comment:
Gelato confirmed that they did this at the same time as the rest of the superchain, `1742486400`